semgrep-core/src/parsing/Parse_rule.ml

```diff
@@ -1103,13 +1106,17 @@ let parse_taint_sanitizer ~(is_old : bool) env (key : key) (value : G.expr) =
     if is_old then parse_formula_old_from_dict else parse_formula_from_dict
   in
   let sanitizer_dict = yaml_to_dict env key value in
+  let sanitizer_mutate =
+    take_opt sanitizer_dict env parse_bool "mutate"
```

Comment:
I'm assuming we're changing this from `"mutate"` to something else

Reply:
Yes, finally it's going to be `taint-by-side-effect`.